src/Teams/Data/ApplicationDbContext.cs

```diff
@@ -14,6 +15,7 @@ public class ApplicationDbContext : IdentityDbContext, IApplicationDbContext
         public DbSet<SingleSelectionQuestion> SingleSelectionQuestions { get; set; }
         public DbSet<MultipleAnswerQuestion> MultipleAnswerQuestions { get; set; }
         public DbSet<ProgramCodeQuestion> ProgramCodeQuestions { get; set; }
+        public DbSet<ProgramTextModel> ProgramTexts { get; set; }
```

Comment:
Let's change the name, so it's "QueuedProgram". This will better reflect the idea that these programs are in queue to be tested.